engines/dfc_provider/app/controllers/dfc_provider/social_medias_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module DfcProvider
+  class SocialMediasController < DfcProvider::ApplicationController
```

Comment:
This looks like a nice extensible way to implement it and handle a changeable list of social media handles. But I wonder if just exposing the flat fields on the enterprise would have been fine?

Reply:
Oh, I just realised that's the built-in DFC way 👍

Reply:
Every object in DFC requires an id and the id must point to an API endpoint representing the object. The only exception are blank nodes like Quantity which don't have an id, just unit and value. But the DFC made SocialMedia a proper object with an id. I guess, in theory, you could ask the API: "which enterprise or person has this Instagram handle?" I don't know if that's in the current DFC data model though.